src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs

```diff
@@ -60,7 +63,25 @@ public static bool HasStarted(HttpContext context, out Activity? aspNetActivity)
     {
         PropagationContext propagationContext = textMapPropagator.Extract(default, context.Request, HttpRequestHeaderValuesGetter);
 
-        Activity? activity = AspNetSource.StartActivity(TelemetryHttpModule.AspNetActivityName, ActivityKind.Server, propagationContext.ActivityContext);
+        KeyValuePair<string, object?>[]? tags = null;
+        if (context.Request?.Unvalidated?.Path is string path)
+        {
+            tags = cachedTagsStorage;
```

Comment:
can this be simplified?

`cachedTagsStorage ??= new KeyValuePair<string, object?>[1];`